specification/compute/resource-manager/Microsoft.Compute/stable/2021-10-01/gallery.json

```diff
@@ -2315,6 +2318,13 @@
           },
           "description": "The target regions where the Image Version is going to be replicated to. This property is updatable."
         },
+        "targetExtendedLocations": {
```

Comment:
@kangsun-ctrl  please put this property last in the list  for that the parameter to go last to avoid breaking change